vignettes/pull_request_comments.Rmd

```diff
@@ -0,0 +1,249 @@
+---
+title: "Download GitHub Pull Request Comments via REST API"
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteIndexEntry{Download GitHub Pull Request Comments via REST API}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+# Introduction
+
+In this vignette, the goal is to understand and visualize comments on
+the GitHub Pull Requests Conversations tab. The goal is to see the
```

Comment:
This is too vague. The goal is to *download* all comments associated to pull requests. Then you explain the visual side briefly as 4 bullet list. Use the naming here consistently with the name of the sub-sections below so the bullets also serve as table of content and the reader knows you are expanding on them below in the subsequent sections. 

When I see the introduction, I need to discern quickly if this is the notebook I am looking for to do something actionable. "visualize" could mean you are plotting them. Your primary goal is to download, parse and combine them so the conversations tab can be *reconstructed* from different endpoints. The default assumption of a user is that one endpoint gives you everything. Clarify that doesn't exist from the get go here.